.github/workflows/stale.yml

```
@@ -14,11 +14,17 @@ jobs:
         with:
           days-before-issue-stale: 730
           days-before-issue-close: 14
-          stale-issue-label: "stale"
+          stale-issue-label: "Stale"
           stale-issue-message: >
             This issue is stale because it has been open for 2 years. It will be closed if no further action occurs in 14 days. 
           close-issue-message: >
```

For a multiline message, this needs to be '|'
```suggestion
          close-issue-message: |
```